src/renderer/components/status-bar/status-bar.test.tsx

```diff
@@ -5,18 +5,16 @@
 
```

Comment:
This test should be written as behaviour. Let's do this together today when you start your workday? :) Shouldn't be that hard to modify and I'll bet we'll catch some bugs while we're doing it.